website/src/docs/installation.md

```diff
@@ -8,15 +8,56 @@ outline: deep
 
 Task offers many installation methods. Check out the available methods below.
 
-::: info
+## Official Package Managers
 
-Some of the methods below are marked as
-![Community](https://img.shields.io/badge/Community%20Owned-orange). This means
-they are not maintained by the Task team and may not be up-to-date.
+These installation methods are maintained by the Task team and are always up-to-date.
 
-:::
+### Fedora ([dnf](https://docs.fedoraproject.org/en-US/quick-docs/dnf)) ![Fedora](https://img.shields.io/badge/Fedora-51A2DA?logo=fedora&logoColor=fff) ![Linux](https://img.shields.io/badge/Linux-FCC624?logo=linux&logoColor=black) {#fedora}
+[[package](https://cloudsmith.io/~task/repos/task/packages/?sort=-format&q=format%3Arpm)]
+
+If you Set up the repository by running :
+
+```shell
+curl -1sLf \
+  'https://dl.cloudsmith.io/public/task/task/setup.rpm.sh' \
+  | sudo -E bash
+```
+Then you can install Task with:
+```shell
+dnf install task
+```
+
+### RedHat / CentOs ([yum](https://docs.fedoraproject.org/en-US/quick-docs/dnf)) ![Fedora](https://img.shields.io/badge/Red_Hat-EE0000?logo=redhat&logoColor=white) ![Linux](https://img.shields.io/badge/Linux-FCC624?logo=linux&logoColor=black) {#redhat}
+
+[[package](https://cloudsmith.io/~task/repos/task/packages/?sort=-format&q=format%3Arpm)]
 
-## Package Managers
+If you Set up the repository by running :
+
+```shell
+curl -1sLf \
+  'https://dl.cloudsmith.io/public/task/task/setup.rpm.sh' \
+  | sudo -E bash
+```
+Then you can install Task with:
+```shell
+yum install task
+```
+
+### Debian / Ubuntu / LinuxMint ([apt](https://doc.ubuntu-fr.org/apt)) ![Ubuntu](https://img.shields.io/badge/Ubuntu-E95420?logo=Ubuntu&logoColor=white) ![Debian](https://img.shields.io/badge/debian-red?logo=debian&logoColor=orange&color=darkred) ![Linux](https://img.shields.io/badge/Linux-FCC624?logo=linux&logoColor=black) {#debian}
+
+[[package](https://cloudsmith.io/~task/repos/task/packages/?sort=-format&q=format%3Adeb)]}
+
+If you Set up the repository by running :
+
+```shell
+curl -1sLf \
+  'https://dl.cloudsmith.io/public/task/task/setup.rpm.sh' \
+  | sudo -E bash
```

Comment:
I wonder if wrapping the command in three lines wouldn't make it possibly error-prone to paste?

Probably not, but I still lean into making it a single line.

Reply:
To be honest, I just copy / past the documentation from Cloudsmith
happy to do it in onliner if you want 
<img width="985" height="311" alt="image" src="https://github.com/user-attachments/assets/28a730e2-57a0-4216-aed0-5cb194f94d9c" />

Reply:
I vote to make them one-liners, yes.